components/Text.js

```diff
@@ -16,6 +16,7 @@ export const P = styled.p.attrs(props => ({
   ${whiteSpace}
   ${wordBreak}
   ${cursor}
+  ${letterSpacing}
```

Comment:
This should not be necessary, `letterSpacing is already included in `typography`: https://styled-system.com/api#typography